scripts/k8s-storage/create-configmap.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+#
+# Create the Ceph-CSI ConfigMap based on the configuration obtained from the
+# Rook deployment.
+#
+# The ConfigMap is referenced in the StorageClasses that are used by
+# driver-*.yaml manifests in the k8s-e2e-external-storage CI job.
+#
+# Requirements:
+# - kubectl in the path
+# - working KUBE_CONFIG either in environment, or default config files
+# - deployment done with Rook
+#
+
+# exit on error
+set -e
+
+TOOLBOX_POD=$(kubectl -n rook-ceph get pods -l app=rook-ceph-tools -o=jsonpath='{.items[0].metadata.name}')
+FS_ID=$(kubectl -n rook-ceph exec "${TOOLBOX_POD}" ceph fsid)
+MONITOR=$(kubectl -n rook-ceph get services -l app=rook-ceph-mon -o=jsonpath='{.items[0].spec.clusterIP}:{.items[0].spec.ports[0].port}')
+
+cat << EOF | kubectl -n rook-ceph create -f -
```

Comment:
configmap needs to be created in the namespace where the cephcsi is running.

Reply:
yes, of course! hmm, it doen't get passed from the job yet, will need to add it there too.